README.md

```diff
@@ -6,6 +6,7 @@
 - Make simple calculations.
 - Read a random math-related quote.
 
+
```

Comment:
- [ ] In `src/redux/books/books.js` reducer kindly ensure there are only two cases: `ADD_BOOK` and `REMOVE_BOOK`. Please remove the `FILTER_BOOKS` case. See project requirement [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_create_and_remove_book.md#:~:text=Write%20your%20reducer%20and%20export%20it%20as%20default%3A).

Reply:
- [ ] OPTIONAL - Also, in the `REMOVE_BOOK` case you can simply use the Array.prototype.filter method to filter out the book containing the payload id.

Reply:
:ok_hand: Thanks for pointing it out. 
- :books:  I have already removed the FILTER_BOOKS case from the books reducer and created a dedicated reducer for it.
- :+1: I have also modified the REMOVE_BOOK to filter items out.